recipes/liblbfgs/build.sh

```diff
@@ -0,0 +1,10 @@
+./configure --prefix $PREFIX 
+make
+make install
+
+mkdir -p $PREFIX/bin
+cp sample/sample $PREFIX/bin
+
+# run test binary 
+# note this seems not to work in meta.yaml test section due to some linking problems
+./sample/sample > /dev/null
```

Comment:
Sorry one last comment, I notice `sample` is being installed into `$PREFIX/bin`. It should probably be removed:

```suggestion
./sample/sample > /dev/null
# Remove the test binary from the installed environment
rm $PREFIX/bin/sample
```

Reply:
true. thanks for all the helpful comments.